### DISCUSSION TITLE
Filtering cams

### DISCUSSION THREAD

**Discussion by PerpSearch** [mrlt8/docker-wyze-bridge#180] [Q&A] [answered]
Let me start out by admitting that there may be a lot going on here, but I'm just beginning to feel my way through this level of tech and I thought I'd start addressing the easier issues first.    I have 6 V3's, 2 V2's the doorbell, and a V3 (pool construction cam) that's shared w/ me.  I'm running a Win10 system with a Synology DS218+ under DSM 7.

1. I wanted to filter or block the shared V3 because having it run 24/7 would be an unnecessary of streaming data. I've tried the FILTER_MODEL variable I read somewhere, but it seems to have no effect. 

2. I have a couple cams that are reported to be "stream is UP!" but I have yet to be able to view.  As in the log, I've used RTSP://192.168.1.76:8554/garage in VLC and I get: _Connection failed: VLC could not connect to "192.168.1.76:8554"._

3. Anything else you see that ought to help.
 
🚀 STARTING DOCKER-WYZE-BRIDGE v0.7.6

2021/10/20 21:40:20 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'user' data

2021/10/20 21:40:20 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'auth' data

2021/10/20 21:40:20 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...

2021/10/20 21:40:21 [WyzeBridge] 💾 Saving 'auth' to local cache...

2021/10/20 21:40:21 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...

2021/10/20 21:40:21 [WyzeBridge] 💾 Saving 'user' to local cache...

2021/10/20 21:40:21 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'cameras' data

2021/10/20 21:40:21 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...

2021/10/20 21:40:22 [WyzeBridge] 💾 Saving 'cameras' to local cache...

2021/10/20 21:40:22 [WyzeBridge] 💔 DTLS on Pan FW:4.49.1.21

2021/10/20 21:40:22 [WyzeBridge] 💔 DTLS on Northwest FW:4.36.6.17

2021/10/20 21:40:22 [WyzeBridge] 💔 DTLS on Northeast FW:4.36.6.17

2021/10/20 21:40:22 [WyzeBridge] 💔 DTLS on Sense FW:4.9.8.352

🎬 STARTING ALL 6 CAMERAS

2021/10/20 21:40:22 [WyzeBridge] Starting rtsp-simple-server v0.17.3

2021/10/20 21:40:23 [North] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:40:23 [North] 👻 Camera offline. WILL retry in 10s.

2021/10/20 21:40:28 [Southwest] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:40:28 [Southwest] 👻 Camera offline. WILL retry in 10s.

2021/10/20 21:40:29 [Front Door] 🎉 Starting HD 120kb/s Stream for WyzeCam DOORBELL in "LAN mode" FW: 4.25.0.267 IP: 192.168.1.92 WiFi: 49%

2021/10/20 21:40:30 [Garage] ☁️ WARNING: Camera is connected via "RELAY mode". Stream may consume additional bandwidth!

2021/10/20 21:40:30 [Garage] 🎉 Starting HD 120kb/s Stream for WyzeCam V2 in "RELAY mode" FW: 4.28.4.41 IP: 192.168.1.76 WiFi: 93%

2021/10/20 21:40:31 [Front Door] WARNING: Received incomplete frame

2021/10/20 21:40:33 [Front Door] WARNING: Lost frame

2021/10/20 21:40:35 [Pool construction camera] AV_ER_TIMEOUT

2021/10/20 21:40:35 [RTSP][FRONT-DOOR] ✅ '/front-door' stream is UP!

2021/10/20 21:40:36 [North] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:40:36 [North] 👻 Camera offline. WILL retry in 20s.

2021/10/20 21:40:36 [RTSP][GARAGE] ✅ '/garage' stream is UP!

2021/10/20 21:40:39 [Southwest] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:40:39 [Southwest] 👻 Camera offline. WILL retry in 20s.

2021/10/20 21:40:39 [Pool construction camera] ☁️ WARNING: Camera is connected via "P2P mode". Stream may consume additional bandwidth!

2021/10/20 21:40:39 [Pool construction camera] 🎉 Starting HD 120kb/s Stream for WyzeCam V3 in "P2P mode" FW: 4.36.3.19 IP: 192.168.4.121 WiFi: 100%

2021/10/20 21:40:41 [Front Door] WARNING: Lost frame

2021/10/20 21:40:44 [RTSP][POOL-CONSTRUCTION-CAMERA] ✅ '/pool-construction-camera' stream is UP!

2021/10/20 21:40:46 [Front Door] WARNING: Received incomplete frame

2021/10/20 21:40:58 [North] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:40:58 [North] 👻 Camera offline. WILL retry in 30s.

2021/10/20 21:41:03 [Southwest] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:41:03 [Southwest] 👻 Camera offline. WILL retry in 30s.

2021/10/20 21:41:28 [Southeast] IOTC_ER_CAN_NOT_FIND_DEVICE

2021/10/20 21:41:28 [Southeast] Camera firmware may be incompatible

2021/10/20 21:41:33 [North] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:41:33 [North] 👻 Camera offline. WILL retry in 40s.

2021/10/20 21:42:14 [Southwest] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:42:14 [Southwest] 👻 Camera offline. WILL retry in 40s.

2021/10/20 21:42:54 [Southwest] IOTC_ER_DEVICE_OFFLINE

2021/10/20 21:42:54 [Southwest] 👻 Camera offline. WILL retry in 50s.

2021/10/20 21:42:54 [Southwest] 👻 Camera offline. WILL retry in 50s.

Thanks for your patience and any advise,
Charley

**Comment by mrlt8** [comment, marked as answer]
1. The filtering mode will default to whitelist mode unless you add `FILTER_BLOCK=true` which will flip it into a blacklist. In your situation, filtering by model would result in blocking all the V3s, so you may want to use `FILTER_NAMES` like:
```yaml
- FILTER_NAMES=pool construction cam
- FILTER_BLOCK=true
```
2. Do you have another service already running on port 8554? Could also test the HLS stream in a browser `http://localhost:8888/cam-nickname`

**Comment by PerpSearch** [comment]
. . . and the pool cam is gone thank you very much!  I struggled a bit because I read somewhere that in multi worded cam names I was supposed to use a '-' but apparently that ain't so.  As to port conflict, I tried HLS and it was a no go too.  I'll try to research the port issue as best I can, but I'm not running any other docker containers and little else w/ the NAS other than file server and back-ups.  That said, I upgraded to the most recent version of Synology DSM and there were port conflicts that broke AdGuard for everyone so it could be DSM 7.  Thanks again.

BTW, this is where I am now.  I filtered a few for now.


🚀 STARTING DOCKER-WYZE-BRIDGE v0.7.6

2021/10/21 03:28:06 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'user' data

2021/10/21 03:28:06 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'auth' data

2021/10/21 03:28:06 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...

2021/10/21 03:28:07 [WyzeBridge] 💾 Saving 'auth' to local cache...

2021/10/21 03:28:07 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...

2021/10/21 03:28:07 [WyzeBridge] 💾 Saving 'user' to local cache...

2021/10/21 03:28:07 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'cameras' data

2021/10/21 03:28:07 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...

2021/10/21 03:28:08 [WyzeBridge] 💾 Saving 'cameras' to local cache...

2021/10/21 03:28:08 [WyzeBridge] 💔 DTLS on Pan FW:4.49.1.21

2021/10/21 03:28:08 [WyzeBridge] 💔 DTLS on Northwest FW:4.36.6.17

2021/10/21 03:28:08 [WyzeBridge] 💔 DTLS on Northeast FW:4.36.6.17

2021/10/21 03:28:08 [WyzeBridge] 💔 DTLS on Sense FW:4.9.8.352

🪄 BLACKLIST MODE ON

🎬 STARTING 2 OF 6 CAMERAS

2021/10/21 03:28:08 [WyzeBridge] Starting rtsp-simple-server v0.17.3

2021/10/21 03:28:12 [Front Door] 🎉 Starting HD 120kb/s Stream for WyzeCam DOORBELL in "LAN mode" FW: 4.25.0.267 IP: 192.168.1.92 WiFi: 42%

2021/10/21 03:28:12 [Garage] 🎉 Starting HD 120kb/s Stream for WyzeCam V2 in "LAN mode" FW: 4.28.4.41 IP: 192.168.1.76 WiFi: 100%

2021/10/21 03:28:13 [Garage] WARNING: Lost frame

2021/10/21 03:28:17 [Front Door] WARNING: Received incomplete frame

2021/10/21 03:28:18 [RTSP][FRONT-DOOR] ✅ '/front-door' stream is UP!

2021/10/21 03:28:20 [RTSP][GARAGE] ✅ '/garage' stream is UP!

Thanks again.

**Reply by mrlt8** [reply]
IIRC, `-` are only used in the URIs for accessing the streams and all other settings should accept a blank space.

I don't have a Synology, but I believe several users are running the bridge on some. Might want to check the port setting in Synology/docker settings to verify that the ports you need are accessible.

**Reply by PerpSearch** [reply]
I'll see what I can figure out. I'm not surprised that there are Synology users out there and that it works for them, but many people have been reluctant to update to DSM version 7 because it breaks things.  I'd love to find one person using 7 that made this work, but I'm still assuming (and hoping) it's me because that maybe fixable whereas not so much with Synology. Thanks again.
